### PR TITLE
Added VI keys (hjkl) for moving and resizing windows

### PR DESCRIPTION
**Pull request by kungfubeaner**
Just a few simple lines of code to be able to move and resize windows using VIM keys.

### CODE REVIEW

**Review by vkareh — CHANGES_REQUESTED** [review]
This builds and works really well, and as a fellow vim user this is pretty useful. However, I'd like other team members to approve before adding this feature and changing behavior.

The change request is to fix the styling. The actual functional code is pretty small, yet this change touches way too many lines that don't belong in it.

File: src/core/keybindings.c [hunk 1]
```diff
@@ -3176,9 +3206,6 @@ handle_toggle_tiled (MetaDisplay *display,
               next_cycle = META_TILE_CYCLE_25;
               break;
             case META_TILE_CYCLE_25:
-              next_cycle = META_TILE_CYCLE_100;
-              break;
-            case META_TILE_CYCLE_100:
```
Comment (vkareh) [inline src/core/keybindings.c:3181]: Why remove this? We just added this recently

Comment (kungfubeaner) [reply src/core/keybindings.c:3181]: I didn't remove it must of happened when I ran the git clone command and was too lazy to start over and copied the file. sorry about that.

Comment (kungfubeaner) [reply src/core/keybindings.c:3181]: I'm new to contributing to other project mainly only used github for personal projects. So yeah I'm a noob at this sorry. Not a noob at coding in C or C++ but at this whole github thing I am.

File: src/core/keybindings.c [hunk 2]
```diff
@@ -1553,13 +1553,15 @@ process_keyboard_move_grab (MetaDisplay *display,
     case XK_KP_Prior:
     case XK_Up:
     case XK_KP_Up:
+	 case XK_k:
```
Comment (vkareh) [inline src/core/keybindings.c:1556]: Much better, thanks - now you still need to adjust the indentation so that it matches the existing code

Comment (kungfubeaner) [reply src/core/keybindings.c:1556]: strange seems even on my end even on my end. i'll see whats going on there.

Comment (kungfubeaner) [reply src/core/keybindings.c]: hmm must be something up with my vim settings seems even on my end but not in the changes

**Review by vkareh — APPROVED** [review]
significantly better, thanks! works well for me

You might want to squash/fixup your commits into a single one (run `git rebase -i HEAD~4` to squash the last 4 commits and follow the instructions in the editor that will pop up: `pick` the first and `fixup` the rest)

**Review by rbuj — CHANGES_REQUESTED** [review]
Wikipedia lists +10 alternatives to the arrow keys, why add the ones used by VIM?
https://en.wikipedia.org/wiki/Arrow_keys
- 5678 keys
- WASD keys
- ESDF keys
- DCAS keys
- IJKL keys
- IJKM keys
- AZ keys
- HJKL keys
- SDF-SPACE and ASD-SPACE
- WQSE keys
- QAOP keys
- ESDX keys

**Comment by kungfubeaner** [comment]
TLDR version:   The rest of the arrow keys don't have a cult following and doesn't become muscle memory.  Most people who use WASD  and the other alternatives would prefer the arrow keys themselves.   For VIM users it becomes almost a necessity.  I'm sure vkareh would vouch for this.


Long version:

Its hard to describe understand unless you experience it yourself after learning and using VIM.  I'm sure vkareh would vouch for this.  When you learn VIM much like typing it becomes muscle memory and it just feels natural after the fact.   Its on the home row keys and it would be inviting for people who use vim.   XFCE already offers these option but for some reason  MATE feels so much more complete and looks a lot nicer imho.   I'm coming from a tiling window (bspwm) manager but I've always felt like something' missing.   I've finally found a desktop environment I like (MATE) and and its 99% perfect.   The only thing missing for perfection is this.  Being able to move and resize windows using VIM keys.  I've been able to make it fit my workflow 100% and even being able to navigate 10 work spaces all with just a keyboard. 

> 
> 
> Wikipedia lists +10 alternatives to the arrow keys, why add the ones used by VIM?
> https://en.wikipedia.org/wiki/Arrow_keys
> 
>     * 5678 keys
> 
>     * WASD keys
> 
>     * ESDF keys
> 
>     * DCAS keys
> 
>     * IJKL keys
> 
>     * IJKM keys
> 
>     * AZ keys
> 
>     * HJKL keys
> 
>     * SDF-SPACE and ASD-SPACE
> 
>     * WQSE keys
> 
>     * QAOP keys
> 
>     * ESDX keys

**Review by rbuj — APPROVED** [review]
It looks good to me even though I'm not a **vi** evangelist.

> HJKL keys are still ubiquitous in newly developed Unix software even though today's keyboards have arrow keys.

https://en.wikipedia.org/wiki/Arrow_keys#HJKL_keys

**Review by raveit65 — APPROVED** [review]
LGTM
